WORKSPACE

```diff
@@ -68,9 +61,12 @@ http_file(
 )
 
 # For Jetty
-http_file(
+new_http_archive(
    name = "jetty",
+   build_file = "BUILD.jetty",
    url = "http://central.maven.org/maven2/org/eclipse/jetty/jetty-distribution/9.4.4.v20170414/jetty-distribution-9.4.4.v20170414.tar.gz",
+   strip_prefix = "jetty-distribution-9.4.4.v20170414/",
```

Comment:
I didn't try, but it would probably be pretty similar. maven_jar just exposes a filegroup that we would need to tar:
https://github.com/bazelbuild/bazel/blob/516f7e9912ad3130ce18322d43be3e2f57c97ca1/tools/build_defs/repo/maven_rules.bzl#L167